### PR TITLE
add sortable tables

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves https://github.com/redpanda-data/documentation-private/issues/2501

Example
https://deploy-preview-193--docs-ui.netlify.app/#tables

### CODE REVIEW

**Comment by JakeSCahill** [comment]
Nice this is a great UX improvement. We should use the pointer cursor on hover. Also I'm seeing some weird behavior with the arrows. Should up/down appear next to each other?

**Comment by Deflaimun** [comment]
From their implementation, up and down appears so you can recognize that the table isn't sorted. We can modify this behavior if it's not helpful. 

Great suggestion on the pointer.

**Comment by JakeSCahill** [comment]
One appears larger than the other even when the columns are not sorted. Then on click only one arrow is visible. If you click another column, both arrows appear again. We should make the experience consistent.

**Comment by Deflaimun** [comment]
Modified so that only one arrow shows up at the same time. Default: up.

File: src/css/vendor/sortable-tables.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+table.sortable th:not(.sorttable_sorted):not(.sorttable_sorted_reverse):not(.sorttable_nosort)::after {
+  content: " \25B4";
+}
+
+table.sortable:hover {
```
Comment (JakeSCahill) [inline src/css/vendor/sortable-tables.css:5]: Can we target the `thead` element? That way the pointer doesn't appear on the whole table

**Review by JakeSCahill — APPROVED** [review]
Really cool

**Pull request merged**